lib/cretonne/meta/cdsl/instructions.py

```diff
@@ -95,6 +97,8 @@ class Instruction(object):
             'is_branch': 'True for all branch or jump instructions.',
             'is_call': 'Is this a call instruction?',
             'is_return': 'Is this a return instruction?',
+            'can_load': 'Can this instruction read from memory?',
+            'can_store': 'Can this instruction write to memory?',
```

Comment:
We should have a way of catching when you use `can_load` in one place and `may_load` in another place.

The `Instruction` constructor should complain about unknown flags.